pkg/rest/apisurface.go

```diff
@@ -171,15 +171,15 @@ func unpackDeprovisionRequest(r *http.Request) (*osb.DeprovisionRequest, error)
 func (s *APISurface) LastOperationHandler(w http.ResponseWriter, r *http.Request) {
 	version := getBrokerAPIVersionFromRequest(r)
 	if err := s.BusinessLogic.ValidateBrokerAPIVersion(version); err != nil {
-		writeError(w, err)
+		writeError(w, err, http.StatusPreconditionFailed)
 		return
 	}
 
 	request, err := unpackLastOperationRequest(r)
 	if err != nil {
 		// TODO: This should return a 400 in this case as it is either
```

Comment:
Guess we can remove these comments now? :)

Reply:
Let's keep them in - they still return the wrong code, I just carried out the mechanical refactor. Let's go through and add integration tests and correct them as we add tests.